README.md

```diff
@@ -665,6 +665,55 @@ spec:
           exporters: [debug]
 ```
 
+#### Using imagePullSecrets
```

Comment:
This is being added under TA. It should rather belong to the general collector section.